sdk/src/client/api/address.rs

```diff
@@ -212,8 +196,10 @@ impl<'a> GetAddressesBuilder<'a> {
                 self.coin_type,
                 self.account_index,
                 self.range,
-                true,
-                self.options.clone(),
+                self.options.map(|mut o| {
+                    o.internal = true;
+                    o
+                }),
```

Comment:
If options is_none() the map will not be used and it will generate public addresses

Reply:
I am très stupide.